src/Components/WebAssembly/testassets/ThreadingApp/Pages/Counter.razor

```diff
@@ -0,0 +1,79 @@
+﻿@page "/counter"
```

Comment:
This seems to be testing several scenarios simultaneously, could we split them in one page per scenario and provide clearer naming?